charts/rstudio-pm/values.yaml

```diff
@@ -88,6 +88,9 @@ rstudioPMKey: false
 command: false
 # -- args is the pod's run arguments. By default, it uses the container's default
 args: false
+# -- container [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) defines privilege and access control settings
+securityContext:
```

Comment:
I've looked at [various](https://github.com/kubernetes/ingress-nginx/blob/24b8e32b088bec15e1f40deab8fefc8bd1b6c2da/charts/ingress-nginx/values.yaml#L825) [community](https://github.com/grafana/helm-charts/blob/f942ba605f8780518b6ac7f52eb68fb275c02e8b/charts/grafana/values.yaml#L101) charts and the convention is to use `containerSecurityContext` for this to disambiguate. Let's follow that convention.